<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CViewerWindow</class>
 <widget class="QMainWindow" name="CViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Map Designer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="StorageMap_View">
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="acNewGraphML"/>
    <addaction name="acLoadGraphML"/>
    <addaction name="acSaveGraphML"/>
    <addaction name="acSaveGraphMLAs"/>
    <addaction name="acFullScreen"/>
    <addaction name="acGenTestGraph"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="acFitToPage"/>
    <addaction name="acZoomIn"/>
    <addaction name="acZoomOut"/>
    <addaction name="acGrid"/>
    <addaction name="acSnapToGrid"/>
    <addaction name="acBBox"/>
   </widget>
   <widget class="QMenu" name="menuGraph_Edit">
    <property name="title">
     <string>Graph Edit</string>
    </property>
    <addaction name="acAddNode"/>
    <addaction name="acAddEdge_direct"/>
    <addaction name="acAddEdge_reverse"/>
    <addaction name="acAddEdge"/>
    <addaction name="acDelMultiEdge"/>
    <addaction name="acReverseEdges"/>
    <addaction name="acSelectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuGraph_Edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolMain">
   <property name="windowTitle">
    <string>Scene tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acFitToPage"/>
   <addaction name="acZoomIn"/>
   <addaction name="acZoomOut"/>
   <addaction name="acGrid"/>
   <addaction name="acSnapToGrid"/>
   <addaction name="acBBox"/>
   <addaction name="acInfoRails"/>
   <addaction name="acMainRail"/>
   <addaction name="acLockEditing"/>
  </widget>
  <widget class="QToolBar" name="toolEdit">
   <property name="windowTitle">
    <string>Graph tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acAddNode"/>
   <addaction name="acAddEdge_direct"/>
   <addaction name="acAddEdge_reverse"/>
   <addaction name="acAddEdge"/>
   <addaction name="acDelMultiEdge"/>
   <addaction name="acReverseEdges"/>
   <addaction name="acAlignVertical"/>
   <addaction name="acAlignHorisontal"/>
  </widget>
  <widget class="QDockWidget" name="dkObjectProps">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Object Props</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTableView" name="tvObjProps">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkView">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Scene preferences</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
         <item>
          <widget class="QLabel" name="lbGridSize">
           <property name="text">
            <string>Grid size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbGridSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>1000000000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolBar" name="toolMain">
         <property name="windowTitle">
          <string>Tool Bar</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <addaction name="acGrid"/>
         <addaction name="acSnapToGrid"/>
         <addaction name="acBBox"/>
         <addaction name="acSpecialLines"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dkObjectWdiget">
   <property name="windowTitle">
    <string>Object Wdiget</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dkObjectWdiget_Contents">
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
  </widget>
  <action name="acFitToPage">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_fit.png</normaloff>:/images/zoom_fit.png</iconset>
   </property>
   <property name="text">
    <string>Fit To Page</string>
   </property>
   <property name="toolTip">
    <string>Fit To Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="acZoomIn">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="acZoomOut">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="acGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/grid.png</normaloff>:/images/grid.png</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>Grid</string>
   </property>
   <property name="shortcut">
    <string>#</string>
   </property>
  </action>
  <action name="acBBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/bbox.png</normaloff>:/images/bbox.png</iconset>
   </property>
   <property name="text">
    <string>BBox</string>
   </property>
   <property name="toolTip">
    <string>BBox</string>
   </property>
  </action>
  <action name="acLoadGraphML">
   <property name="text">
    <string>Load GraphML</string>
   </property>
   <property name="toolTip">
    <string>Load GraphML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="acSaveGraphMLAs">
   <property name="text">
    <string>Save GraphML as...</string>
   </property>
   <property name="toolTip">
    <string>Save GraphML as..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="acAddNode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_node.png</normaloff>:/images/add_node.png</iconset>
   </property>
   <property name="text">
    <string>Add node mode</string>
   </property>
   <property name="toolTip">
    <string>Add node mode</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="acAddEdge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge.png</normaloff>:/images/add_edge.png</iconset>
   </property>
   <property name="text">
    <string>Add edges</string>
   </property>
   <property name="toolTip">
    <string>Add edges for selected nodes</string>
   </property>
  </action>
  <action name="acAddEdge_direct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge_1-2_off.png</normaloff>
     <normalon>:/images/add_edge_1-2.png</normalon>:/images/add_edge_1-2_off.png</iconset>
   </property>
   <property name="text">
    <string>Add edges direct mode</string>
   </property>
   <property name="toolTip">
    <string>Add edges in a direct order (1 --&gt; 2)</string>
   </property>
  </action>
  <action name="acAddEdge_reverse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/add_edge_2-1_off.png</normaloff>
     <normalon>:/images/add_edge_2-1.png</normalon>:/images/add_edge_2-1_off.png</iconset>
   </property>
   <property name="text">
    <string>Add edges reverse mode</string>
   </property>
   <property name="toolTip">
    <string>Add edges in a reverse order (1 &lt;-- 2)</string>
   </property>
  </action>
  <action name="acDelMultiEdge">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/del_edge.png</normaloff>:/images/del_edge.png</iconset>
   </property>
   <property name="text">
    <string>Delete multiple edge</string>
   </property>
   <property name="toolTip">
    <string>Delete multiple edge</string>
   </property>
  </action>
  <action name="acReverseEdges">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/reverse_edge.png</normaloff>:/images/reverse_edge.png</iconset>
   </property>
   <property name="text">
    <string>Reverse selected edges</string>
   </property>
   <property name="toolTip">
    <string>Reverse selected edges</string>
   </property>
  </action>
  <action name="acSnapToGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/snap_to_grid.png</normaloff>:/images/snap_to_grid.png</iconset>
   </property>
   <property name="text">
    <string>Snap to grid</string>
   </property>
   <property name="toolTip">
    <string>Snap to grid</string>
   </property>
  </action>
  <action name="acInfoRails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/info_rails.png</normaloff>:/images/info_rails.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide info rails</string>
   </property>
   <property name="toolTip">
    <string>Show/hide info rails</string>
   </property>
  </action>
  <action name="acMainRail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/main_rail.png</normaloff>:/images/main_rail.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide main rail</string>
   </property>
   <property name="toolTip">
    <string>Show/hide main rail</string>
   </property>
  </action>
  <action name="acSelectAll">
   <property name="text">
    <string>Select all items</string>
   </property>
   <property name="toolTip">
    <string>Select all items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="acSaveGraphML">
   <property name="text">
    <string>Save GraphML</string>
   </property>
   <property name="toolTip">
    <string>Save GraphML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="acNewGraphML">
   <property name="text">
    <string>New GraphML</string>
   </property>
   <property name="toolTip">
    <string>New GraphML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="acLockEditing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/lock_editing_off.png</normaloff>
     <normalon>:/images/lock_editing_on.png</normalon>:/images/lock_editing_off.png</iconset>
   </property>
   <property name="text">
    <string>Lock editing</string>
   </property>
   <property name="toolTip">
    <string>Lock scene editing</string>
   </property>
  </action>
  <action name="acSpecialLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/special_lines.png</normaloff>:/images/special_lines.png</iconset>
   </property>
   <property name="text">
    <string>Storages rotate lines</string>
   </property>
   <property name="toolTip">
    <string>Show storages rotate lines</string>
   </property>
  </action>
  <action name="acFullScreen">
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="toolTip">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="acGenTestGraph">
   <property name="text">
    <string>Generate test graph</string>
   </property>
   <property name="toolTip">
    <string>Generate test graph</string>
   </property>
  </action>
  <action name="acAlignVertical">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/align_v.png</normaloff>:/images/align_v.png</iconset>
   </property>
   <property name="text">
    <string>Align vertical</string>
   </property>
   <property name="toolTip">
    <string>Align nodes vertically, base node - first selected node.</string>
   </property>
  </action>
  <action name="acAlignHorisontal">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/align_h.png</normaloff>:/images/align_h.png</iconset>
   </property>
   <property name="text">
    <string>Align horisontal</string>
   </property>
   <property name="toolTip">
    <string>Align nodes horisontally, base node - first selected node.</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
